clang-tools-extra/test/clang-doc/mustache-separate-namespace.cpp

```diff
@@ -13,3 +14,17 @@ namespace MyNamespace {
 // CHECK-NEXT:        </a>
 // CHECK-NEXT:    </li>
 // CHECK-NEXT: </ul>
+
+// COM: Check that the empty global namespace doesn't contain tag mismatches.
+// CHECK-GLOBAL:             <main>
+// CHECK-GLOBAL-NEXT:            <div class="container">
+// CHECK-GLOBAL-NEXT:                <div class="sidebar">
+// CHECK-GLOBAL-NEXT:                    <h2> </h2>
+// CHECK-GLOBAL-NEXT:                    <ul>
+// CHECK-GLOBAL-NEXT:                    </ul>
+// CHECK-GLOBAL-NEXT:                </div>
+// CHECK-GLOBAL-NEXT:                <div class="resizer" id="resizer"></div>
+// CHECK-GLOBAL-NEXT:                <div class="content">
+// CHECK-GLOBAL:                     </div>
```

Comment:
Is this to deal w/ the empty line? we have CHECK-EMPTY for that, and the next item can be CHECK-NEXT.   Otherwise yo may need a CHECK-NOT to ensure it isn't in the output, since it would just be skipped/ignored.

Reply:
Thanks, that helps.